frontend/src/components/shared/FullScreenWrapper/FullScreenWrapper.tsx

```diff
@@ -55,9 +55,9 @@ interface State {
  * to fixed and cover all screen, updating wrapped element height and width
  */
 class FullScreenWrapper extends PureComponent<Props, State> {
-  static contextType = AppContext
+  declare context: React.ContextType<typeof AppContext>
```

Comment:
This lets the typechecker know that `this.context` is an `AppContext` (as opposed to an `any`)